### PR TITLE
feat(license-service): add Scopes

### PR DESCRIPTION
**Pull request by osk**
# License service: add Scopes

Adds scopes to the generic license service:

* `@island.is/internal`, shared generic scope for APIs until an explicit scope is needed
* `@island.is/licenses:verify`, can verify licenses

Scopes have only been created in **dev environment** and attched to clients:

* `@island.is/internal` => `island-is-1`, `@island.is/app`, and `@island.is/apps/scanner`
* `@island.is/licenses:verify` => `@island.is/app`, `@island.is/apps/scanner`

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] ~I have added tests that prove my fix is effective or that my feature works~
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/auth/scopes/src/lib/licenses.scope.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export enum LicenseScope {
+  // Client can read all licenses
+  read = '@island.is/licenses:read',
+
+  // Client can generate licenses to display (e.g. pkpass for drivers license)
+  generate = '@island.is/licenses:generate',
```
Comment (eirikurn) [inline libs/auth/scopes/src/lib/licenses.scope.ts:6]: The plan is to start with a shared `@island.is/internal` scope for most APIs until we have a use case for explicit scopes (either delegations or external clients). 

I don't know ... `GenericScope.internal`? :)

**Review by eirikurn — CHANGES_REQUESTED** [review]

File: libs/auth/scopes/src/lib/licenses.scope.ts (hunk 1 above)
Comment (osk) [reply libs/auth/scopes/src/lib/licenses.scope.ts:6]: OK, will add, the generic scope, name sounds fine.

**Review by eirikurn — APPROVED** [review]
Great! Let's get these scopes created and added to the web and app clients.

**Comment by osk** [comment]
Old and outdated

**Pull request closed without merging**